### ISSUE TITLE
parallel_testrun_count throws "comparison of String with 0 failed" error

### ISSUE THREAD

**Issue by githubist16** [lyndsey-ferguson/fastlane-plugin-test_center#185]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
ran multiscan with the parallel_testrun_count set to a number and got the following error:
[00:03:16]: ------------------------
[00:03:16]: --- Step: multi_scan ---
[00:03:16]: ------------------------
[00:03:16]: Error setting value '2' for option 'parallel_testrun_count'
[00:03:16]: You passed invalid parameters to 'multi_scan'.
[00:03:16]: Check out the error below and available options by running `fastlane action multi_scan`

[!] comparison of String with 0 failed

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<details>
  <pre>[00:06:58]: Generating fastlane environment output, this might take a few seconds...
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                             |
| --------------------------- | ------------------------------------------------- |
| OS                          | 10.14.6                                           |
| Ruby                        | 2.4.4                                             |
| Bundler?                    | true                                              |
| Git                         | git version 2.20.1 (Apple Git-117)                |
| Installation Source         | ~/.rvm/gems/ruby-2.4.4@earlgrey_gems/bin/fastlane |
| Host                        | Mac OS X 10.14.6 (18G103)                         |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.4.4/lib                      |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                       |
| Is contained                | false                                             |
| Is homebrew                 | false                                             |
| Is installed via Fabric.app | false                                             |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/       |
| Xcode Version               | 10.3                                              |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.138.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                      | Version | Update-Status |
| --------------------------- | ------- | ------------- |
| fastlane-plugin-test_center | 3.8.16  | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                         | Version      |
| --------------------------- | ------------ |
| did_you_mean                | 1.1.0        |
| bundler                     | 1.17.3       |
| CFPropertyList              | 2.3.6        |
| concurrent-ruby             | 1.1.5        |
| i18n                        | 0.9.5        |
| minitest                    | 5.13.0       |
| thread_safe                 | 0.3.6        |
| tzinfo                      | 1.2.6        |
| activesupport               | 4.2.11.1     |
| public_suffix               | 2.0.5        |
| addressable                 | 2.7.0        |
| atomos                      | 0.1.3        |
| babosa                      | 1.0.3        |
| claide                      | 1.0.3        |
| clamp                       | 0.6.5        |
| fuzzy_match                 | 2.0.4        |
| nap                         | 1.1.0        |
| cocoapods-core              | 1.7.5        |
| cocoapods-deintegrate       | 1.0.4        |
| cocoapods-downloader        | 1.3.0        |
| cocoapods-plugins           | 1.0.0        |
| cocoapods-search            | 1.0.0        |
| cocoapods-stats             | 1.1.0        |
| netrc                       | 0.11.0       |
| cocoapods-trunk             | 1.4.1        |
| cocoapods-try               | 1.1.0        |
| colored2                    | 3.1.2        |
| escape                      | 0.0.4        |
| fourflusher                 | 2.3.1        |
| gh_inspector                | 1.1.3        |
| molinillo                   | 0.6.6        |
| ruby-macho                  | 1.4.0        |
| nanaimo                     | 0.2.6        |
| xcodeproj                   | 1.14.0       |
| cocoapods                   | 1.7.5        |
| colored                     | 1.2          |
| colorize                    | 0.8.1        |
| highline                    | 1.7.10       |
| commander-fastlane          | 4.4.6        |
| declarative                 | 0.0.10       |
| declarative-option          | 0.1.0        |
| digest-crc                  | 0.4.1        |
| unf_ext                     | 0.0.7.6      |
| unf                         | 0.1.4        |
| domain_name                 | 0.5.20190701 |
| dotenv                      | 2.7.5        |
| emoji_regex                 | 1.0.1        |
| excon                       | 0.71.1       |
| multipart-post              | 2.0.0        |
| faraday                     | 0.17.1       |
| http-cookie                 | 1.0.3        |
| faraday-cookie_jar          | 0.0.6        |
| faraday_middleware          | 0.13.1       |
| fastimage                   | 2.1.7        |
| jwt                         | 2.1.0        |
| memoist                     | 0.16.2       |
| multi_json                  | 1.14.1       |
| os                          | 1.0.1        |
| signet                      | 0.12.0       |
| googleauth                  | 0.6.7        |
| httpclient                  | 2.8.3        |
| mime-types-data             | 3.2019.1009  |
| mime-types                  | 3.3.1        |
| uber                        | 0.1.0        |
| representable               | 3.0.4        |
| retriable                   | 3.1.2        |
| google-api-client           | 0.23.9       |
| google-cloud-env            | 1.3.0        |
| google-cloud-core           | 1.4.1        |
| google-cloud-storage        | 1.16.0       |
| json                        | 2.0.4        |
| mini_magick                 | 4.9.5        |
| multi_xml                   | 0.6.0        |
| plist                       | 3.5.0        |
| rubyzip                     | 1.3.0        |
| security                    | 0.1.3        |
| naturally                   | 2.2.0        |
| simctl                      | 1.6.7        |
| slack-notifier              | 2.3.2        |
| terminal-notifier           | 2.0.0        |
| unicode-display_width       | 1.6.0        |
| terminal-table              | 1.8.0        |
| tty-screen                  | 0.7.0        |
| tty-cursor                  | 0.7.0        |
| tty-spinner                 | 0.9.2        |
| word_wrap                   | 1.0.0        |
| rouge                       | 2.0.7        |
| xcpretty                    | 0.3.0        |
| xcpretty-travis-formatter   | 1.0.0        |
| xctest_list                 | 1.1.8        |
| fastlane-plugin-test_center | 3.8.16       |
| mini_portile2               | 2.3.0        |
| nokogiri                    | 1.8.5        |
| rest-client                 | 2.0.2        |
| slather                     | 2.4.6        |
| xcov                        | 1.4.3        |
</details>

*generated on:* **2019-12-30**
</pre>
</details>

**Comment by githubist16** [comment]
To resolve issue, updated this file:
/lib/fastlane/plugin/test_center/actions/multi_scan.rb

by adding the type as shown below:

before:
<pre>FastlaneCore::ConfigItem.new(
            key: :parallel_testrun_count,
            description: 'Run simulators each batch of tests and/or each test target in parallel on its own Simulator',
            optional: true,
            is_string: false,
            default_value: 1,
            verify_block: proc do |count|
              UI.user_error!("Error: :parallel_testrun_count must be greater than zero") unless count > 0
              UI.important("Warning: the CoreSimulatorService may fail to connect to simulators if :parallel_testrun_count is greater than 6") if count > 6
            end
          )</pre>

after:
<pre> FastlaneCore::ConfigItem.new(
            key: :parallel_testrun_count,
            description: 'Run simulators each batch of tests and/or each test target in parallel on its own Simulator',
            type:Integer,
            optional: true,
            is_string: false,
            default_value: 1,
            verify_block: proc do |count|
              UI.user_error!("Error: :parallel_testrun_count must be greater than zero") unless count > 0
              UI.important("Warning: the CoreSimulatorService may fail to connect to simulators if :parallel_testrun_count is greater than 6") if count > 6
            end
          )</pre>

**Comment by lyndsey-ferguson** [comment]
Just provide the value `2`, not `'2'` and this problem won't occur.

**Comment by githubist16** [comment]
tried both ways didnt work. Like i mentioned added that line of code to make it work,

**Comment by lyndsey-ferguson** [comment]
It works fine for me, can you post a copy of how you are calling `multi_scan`?
